### PR TITLE
[RHPAM-4045] - Missing description in KieApp for Kie Server persistRe…

### PR DESCRIPTION
**Pull request by spolti**
…pos parameters
See: https://issues.redhat.com/browse/RHPAM-4045

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Comment by spolti** [comment]
@jakubschwan @desmax74 please reivew, once it is approved will cherry-pick to master.

File: pkg/apis/app/v2/kieapp_types.go [hunk 1]
```diff
@@ -129,17 +129,21 @@ type KieServerSet struct {
...
-	ServersKiePvSize string           `json:"serversKiePvSize,omitempty"`
+	ID           string             `json:"id,omitempty"`
+	From         *ImageObjRef       `json:"from,omitempty"`
+	Build        *KieAppBuildObject `json:"build,omitempty"` // S2I Build configuration
+	SSOClient    *SSOAuthClient     `json:"ssoClient,omitempty"`
+	KieAppObject `json:",inline"`
+	Database     *DatabaseObject  `json:"database,omitempty"`
+	Jms          *KieAppJmsObject `json:"jms,omitempty"`
+	Jvm          *JvmObject       `json:"jvm,omitempty"`
+	// PersistRepos enables persistent volume for KIE Server's kie and maven repositories
```
Comment (jakubschwan) [inline pkg/apis/app/v2/kieapp_types.go:140]:
```suggestion
	// PersistRepos enables persistent volumes for KIE Server's kie and maven repositories
```

**Review by jakubschwan — APPROVED** [review]
lgtm, added on small sugestion

**Review by desmax74 — APPROVED** [review]

**Pull request merged**